scylla-cql/src/frame/response/result.rs

```diff
@@ -53,13 +53,15 @@ pub struct TableSpec<'a> {
 pub enum ColumnType<'frame> {
     Native(NativeType),
     Collection {
+        frozen: bool,
         type_: CollectionType<'frame>,
     },
     Vector {
         type_: Box<ColumnType<'frame>>,
         dimensions: u16,
     },
     UserDefinedType {
+        frozen: bool,
         definition: Arc<UserDefinedType<'frame>>,
     },
     Tuple(Vec<ColumnType<'frame>>),
```

Comment:
:wrench: As this is a `pub` type, I suggest adding docstrings to all variants and their fields. Especially the `frozen` flag may require explanation, as it's only meaningful in the schema metadata case, which is not obvious.

Reply:
Added documentation in a new commit.